operator/cmd/mesh/operator-init.go

```diff
@@ -163,7 +163,7 @@ func applyManifest(manifestStr, componentName string, opts *kubectlcmd.Options,
 		l.LogAndFatal(err.Error())
 	}
 	if opts.Wait {
-		err := manifest.WaitForResources(objs, clientSet, opts.WaitTimeout, opts.DryRun, l)
+		err := manifest.WaitForResources(objs, clientSet, opts.WaitTimeout, opts.DryRun, nil)
```

Comment:
this may not be required anymore if you are passing in the wait flag to processmanifest.

Reply:
are you suggesting remove? I removed this, so if not let me know